site_files/site.NCAR

```diff
@@ -1,61 +1,45 @@
-set (APPLE_THIRD_PARTY_DIR "/glade/p/VAST/VAPOR/third-party/apps-2017/Darwin_x86_64")
-set (LINUX_THIRD_PARTY_DIR "/glade/p/VAST/VAPOR/third-party/apps-2017/Linux_x86_64")
-set (WINDOWS_THIRD_PARTY_DIR "C:/Vapor3rdParty2015")
-
-
-if (HOSTNAME MATCHES "yslogin[0-9]+" OR HOSTNAME MATCHES "geyser[0-9]+" OR HOSTNAME MATCHES "caldera[0-9]+")
-	set (ON_GEYSER_OR_CALDERA TRUE)
-else ()
-	set (ON_GEYSER_OR_CALDERA FALSE)
-endif ()
-
-
-# This file is only read if on an NCAR build system
-if (APPLE AND NOT EXISTS "${APPLE_THIRD_PARTY_DIR}")
-	return ()
-elseif (UNIX AND NOT APPLE AND NOT EXISTS "${LINUX_THIRD_PARTY_DIR}" AND NOT ${ON_GEYSER_OR_CALDERA})
-	return ()
-elseif (WIN32 AND NOT EXISTS "${WINDOWS_THIRD_PARTY_DIR}")
-	return ()
-endif ()
-
-
 if (APPLE)
-	set (THIRD_PARTY_DIR "${APPLE_THIRD_PARTY_DIR}")
+    set (THIRD_PARTY_DIR     /usr/local/VAPOR-Deps/2019-Aug			     )
+    set (QTDIR               /usr/local/VAPOR-Deps/2019-Aug/Qt/5.12.4/clang_64       )
+    set (CMAKE_PREFIX_PATH   ${QTDIR}/lib/cmake                                      )
 elseif (UNIX AND NOT APPLE)
-	set (THIRD_PARTY_DIR "${LINUX_THIRD_PARTY_DIR}")
+    set (THIRD_PARTY_DIR     /usr/local/VAPOR-Deps/2019-Aug			     )
+    set (QTDIR                /usr/local/VAPOR-Deps/2019-Aug/Qt/5.12.4/gcc_64)
+    set (CMAKE_PREFIX_PATH    ${QTDIR}/lib/cmake )
+    set (Qt5Core_DIR          ${QTDIR}/lib/cmake/Qt5Core )
+    set (QT_QMAKE_EXECUTABLE  ${QTDIR}/gcc_64/bin/qmake )
 else ()
-	set (THIRD_PARTY_DIR "${WINDOWS_THIRD_PARTY_DIR}")
+	set (THIRD_PARTY_DIR    C:/2019-Aug)
+    set (QTDIR              ${THIRD_PARTY_DIR}/Qt5_12_4/5.12.4/msvc2015_64)
+    set (CMAKE_PREFIX_PATH  ${QTDIR}/lib/cmake/Qt5) 
 endif ()
 
+find_package(Qt5 COMPONENTS Core Widgets REQUIRED)
```

Comment:
This should not be done in the site file.

Reply:
Good catch, thank you.